scripts/run-bundled-codegen.sh

```diff
@@ -0,0 +1,64 @@
+# Bail if errors occur
+set -e
+
+# Get the path to the script directory
+SCRIPT_DIR="$(eval dirname $0)"
+
+# Get the SHASUM of the tarball
+ZIP_FILE="${SCRIPT_DIR}/apollo.tar.gz"
+SHASUM="$(eval /usr/bin/shasum -a 256 ${ZIP_FILE})"
+SHASUM_FILE="${SCRIPT_DIR}/apollo/.shasum"
+APOLLO_DIR="${SCRIPT_DIR}"/apollo
+
+# Helper functions
+remove_existing_apollo() {
+  rm -r "${APOLLO_DIR}"
+}
+
+extract_cli() {
+  tar xvzf "${SCRIPT_DIR}"/apollo.tar.gz -C "${SCRIPT_DIR}"
```

Comment:
Nice for debugging, but I'll note that `tar` extraction _with_ the `-v`(erbose) flag much slower due to STDOUT buffering.  Though if really quantifying if the extraction of _x_ files happened, this might be a good idea to keep!

(Another less noisy but possibly still performant and helpful option might be to print the resulting folder size in KBytes with `du -ks folder/`. 🤷‍♂)

Reply:
I think something was fucking up without the V before but that might have been something else - hold plz...

Reply:
yep didn't need it. removed